test/unit/vl-map-features-layer.test.html

```diff
@@ -198,6 +198,80 @@
         assert.isNull(layer.features[0].getStyle());
         assert.isNull(layer.features[1].getStyle());
       });
+
+      test('kan programmatorisch features verwijderen', async () => {
+        const map = fixture('vl-map-fixture');
+        const layer = map.querySelector('vl-map-features-layer');
+        await map.ready;
+        assert.equal(layer.layer.getSource().getFeatures().length, 1);
```

Comment:
Huidige code is niet verkeerd, maar de leesbaarheid verhoogt misschien door gebruik van `assert.lengthOf` te gebruiken.